pkg/authorization/registry/resourceaccessreview/rest_test.go

```diff
@@ -35,13 +35,13 @@ func (a *testAuthorizer) Authorize(attributes kauthorizer.Attributes) (allowed k
 	// allow the initial check for "can I run this RAR at all"
 	if attributes.GetResource() == "localresourceaccessreviews" {
 		if len(a.deniedNamespaces) != 0 && a.deniedNamespaces.Has(attributes.GetNamespace()) {
-			return kauthorizer.DecisionDeny, "denied initial check", nil
+			return kauthorizer.DecisionNoOpinion, "denied initial check", nil
```

Comment:
Are you sure you do not want to also change the message given you are not firmly denying anymore ?
Also seen elsewhere, question valid for all occurrences

Reply:
> Are you sure you do not want to also change the message given you are not firmly denying anymore ?
> Also seen elsewhere, question valid for all occurrences

No strong opinion.